paraview: update to 5.7.0.

anders-dc opened the pull request.

anders-dc commented:
For some reason `[ci-skip]` is not respected.

jnbr commented:
`[ci skip]` has to be in the commit message to be effective

jnbr commented:
For packages that can not be built by travis, a comment whether a local build was done and if so, for which architectures, is helpful.

anders-dc commented:
Thanks @jnbr. I built it locally on x86_64 without issues. I anticipate that it will build on all platforms that were supported in the previous version 5.6.1. The x64_64_musl version was previously reported to work (#13879).

jnbr commented:
Thanks for the feedback. I'll try a local build for all targets here and merge afterwards.

jnbr commented:
this needs some adjustments:
- musl builds fail due to missing execinfo.h
- i686 build fails because g++ runs out of memory, as a workaround disable debug:
```
if [ "$XBPS_WORDSIZE" -eq 32 ]; then
	nodebug=yes
fi
```

anders-dc commented:
thanks again @jnbr 

> * musl builds fail due to missing execinfo.h
ok, based on the info in the xbps manual I added `archs="~*-musl" in 3d166e0

> * i686 build fails because g++ runs out of memory, as a workaround disable debug:
> 
> ```
> if [ "$XBPS_WORDSIZE" -eq 32 ]; then
> 	nodebug=yes
> fi
> ```

added acordingly in 3d166e0

Johnnynator commented:
The preferred way would be to fix the issue instead of setting it to broken.

You could add something like this in `post_patch`
```bash
post_patch() {
        if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
                vsed -e 's/\(LOGURU_STACKTRACES\) 1/\1 0/' \
                        -i VTK/ThirdParty/loguru/vtkloguru/loguru.cpp
        fi
}
```

anders-dc commented:
@Johnnynator thanks! I removed the `archs` restriction and added the suggested `post_patch()` function in 2a1b945.

anders-dc commented:
I must concede that debugging the musl issues is outside my grasp without an musl machine of my own. My only option would be to force-push to this fork and react to the Travis build log. However, that is a slow and tiresome process. I suggest reinstating the `archs="~*-musl"` option until someone is willing to step in.

abenson commented:
No movement, submitter is gone
